__tests__/cache-save.test.ts

```diff
@@ -70,6 +71,9 @@ describe('run', () => {
 
     // utils
     getCommandOutputSpy = jest.spyOn(utils, 'getCommandOutput');
+
```

Comment:
Same as above comment. Is this change required here

Reply:
Done. Thank you.